src/esp/physics/CollisionGroupHelper.h

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Facebook, Inc. and its affiliates.
+// This source code is licensed under the MIT license found in the
+// LICENSE file in the root directory of this source tree.
+
+#ifndef ESP_PHYSICS_COLLISIONGROUPHELPER_H_
+#define ESP_PHYSICS_COLLISIONGROUPHELPER_H_
+
+namespace esp {
+namespace physics {
+
+/*
+  For reference, here are the standard Bullet groups. It's probably good to
+  stay somewhat consistent with them, in case users are already using them.
+        enum CollisionFilterGroups
+        {
+                DefaultFilter = 1,
+                StaticFilter = 2,
+                KinematicFilter = 4,
+                DebrisFilter = 8,
+                SensorTrigger = 16,
+                CharacterFilter = 32,
+                AllFilter = -1  //all bits sets: DefaultFilter | StaticFilter |
+  KinematicFilter | DebrisFilter | SensorTrigger
+        };
+*/
+
+enum class CollisionGroup {
```

Comment:
This list is specific to Andrew's Fetch rearrangement project, would you agree? Do you expect that hab-sim users need to always fork hab-sim for their application?

I suggest that we add python bindings to let the application specify collision groups as well as their interactions, rather than hard-coding these.

CollisionGroupHelper is a class I created as a temporary hack. I don't think it's suited to merge to master as-is.